css/default.css

```diff
@@ -1044,7 +1044,9 @@ ul.mysidebar li.folder.innerfolder.sidebarcategory .sidebarbutton {
 .language-http,
 .language-gradle,
 .language-css,
-.language-objc {
+.language-objc,
+.language-plaintext,
+.language-text {
```

Comment:
Only comment: Do we need to maintain these manually? (Curious to learn more.)

Reply:
please look via inspect how it fixed the problem with the copy icon bug. by adding the missing classes it fixed the problem.